src/client.rs

```diff
@@ -317,27 +319,37 @@ impl fmt::Debug for RequestBuilder {
 }
 
 /// A Response to a submitted `Request`.
+#[derive(Debug)]
```

Comment:
I'd actually keep the previous `Debug` version, it provides just the information that someone would want when writing logs.